databricks/koalas/series.py

```diff
@@ -5388,6 +5388,103 @@ def argmin(self):
             scol_for(sdf, self._internal.data_spark_column_names[0]) == min_value[0]
         ).head()[0]
 
+    # TODO: Remove skip after dropping Python 3.5
```

Comment:
I'm still not sure why we should skip with py3.5?

Reply:
Sorry we don't need to skip Python3.5.

I'll remove this comment, too.

Reply:
Got it.